app.py

```diff
@@ -111,14 +111,14 @@ def graph_refresh_solver() -> None:
         _THOTH_GRAPH_REFRESH_EAGER_STOP,
     )
 
-    indexes = list(_GRAPH_DB.get_python_package_index_urls())
+    indexes = list(_GRAPH_DB.get_python_package_index_urls_all())
 
     packages = []
     # Iterate over all registered solvers and gather packages which were not solved by them.
     for solver_name in _OPENSHIFT.get_solver_names():
         _LOGGER.info("Checking unsolved packages for solver %r", solver_name)
         solver_info = _GRAPH_DB.parse_python_solver_name(solver_name)
-        for package_name, version, index_url in _GRAPH_DB.get_unsolved_python_package_versions(
```

Comment:
@bissenbay , look at the for loop it , loops through package_name, version, index_url of tuple values in a list,  which is given by this https://github.com/thoth-station/storages/blob/a82d77935510cbde18f3a67c70ed8dd47ef12db7/thoth/storages/graph/postgres.py#L1589
the `get_unsolved_python_package_versions_count` output is a dict which doesn't fit.
WDYT?

Reply:
> @bissenbay , look at the for loop it , loops through package_name, version, index_url of tuple values in a list, which is given by this https://github.com/thoth-station/storages/blob/a82d77935510cbde18f3a67c70ed8dd47ef12db7/thoth/storages/graph/postgres.py#L1589
> the `get_unsolved_python_package_versions_count` output is a dict which doesn't fit.
> WDYT?

But, you can loop the keys of the dict, right :smiley: ? Sure, I'll change them.

Reply:
@bissenbay true, you don't have to change, use the correct one. @pacospace i don't know which one to use, as both would.

Reply:
> @bissenbay true, you don't have to change, use the correct one. @pacospace i don't know which one to use, as both would.

Changed already, :smiley: ! :+1:

Reply:
Take a look at the purpose of the function - it's scheduling unsolved python packages, which corresponds to `get_unsolved_python_package_versions_all`.

Both versions would work though as the dict (no method is called on it) would be expanded to keys:

```python
>>> for i in {"foo": 1, "bar": 2}:
...     print(i)
... 
foo
bar
```

I agree the count method is confusing here. @pacospace any reason why do we have it? IMHO it will return always a dict with values of 0 and 1?

Reply:
The functions to be used are: `get_unsolved_python_package_versions_all`, `get_python_package_index_urls_all`, `get_unanalyzed_python_package_versions_all' as they are giving back `List[str]`

Reply:
Sorry if it is not perfectly clear :disappointed: . There is a card about converting the naming convention document into developer guide and in docs for the website :) I put explanations and examples there :)